pkg/mount/mounter_linux.go

```diff
@@ -13,14 +13,16 @@ const (
 
 	// broflags is the combination of bind and read only
 	broflags = unix.MS_BIND | unix.MS_RDONLY
+
+	none = "non"
```

Comment:
Typo: the e is missing

Reply:
Fixed in https://github.com/containers/storage/pull/523